### PR TITLE
[16.0][MIG] sale_pricelist_from_commitment_date: Migration to 16.0

### PR DESCRIPTION
**Pull request by santostelmo**
Migration of sale_pricelist_from_commitment_date to 16.0

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by cyrilmanuel — APPROVED** [review]

File: sale_pricelist_from_commitment_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Pricelist From Commitment Date",
+    "summary": "Use sale order commitment date to compute line price from pricelist",
+    "version": "16.0.1.0.2",
```
Comment (simahawk) [inline sale_pricelist_from_commitment_date/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by simahawk — APPROVED** [review]
@santostelmo can you fix the version?

**Comment by santostelmo** [comment]
> @santostelmo can you fix the version?

Version fixed

**Pull request merged**